### PR TITLE
Prepare offer event handler tests

### PR DESCRIPTION
**Pull request by kamilbaczek**

### CODE REVIEW

File: Src/Fitnet.IntegrationTests/Common/TestEngine/IntegrationEvents/Handlers/WebApplicationFactoryExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using MediatR;
+using SuperSimpleArchitecture.Fitnet.Shared.Events;
+
+internal static class WebApplicationFactoryExtensions
+{
+    internal static IIntegrationEventHandler<TIntegrationEvent> GetIntegrationEventHandler<TIntegrationEvent>(
+        this WebApplicationFactory<Program> applicationInMemoryFactory)
+        where TIntegrationEvent : IIntegrationEvent
+    {
+        var integrationEventConsumer = applicationInMemoryFactory.Services.GetRequiredService<INotificationHandler<TIntegrationEvent>>();
```
Comment (meaboutsoftware) [inline Src/Fitnet.IntegrationTests/Common/TestEngine/IntegrationEvents/Handlers/WebApplicationFactoryExtensions.cs:12]: Fix formatting (eveything after '=' in the new line)

File: Src/Fitnet.IntegrationTests/Offers/Preapare/PassExpiredEventFaker.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+namespace SuperSimpleArchitecture.Fitnet.IntegrationTests.Offers.Preapare;
```
Comment (meaboutsoftware) [inline Src/Fitnet.IntegrationTests/Offers/Preapare/PassExpiredEventFaker.cs:1]: Namespace should be Prepare, not Pr**ea**pare.

Please adjust it in other places as well

**Review by meaboutsoftware — CHANGES_REQUESTED** [review]

File: Src/Fitnet.IntegrationTests/Offers/Preapare/PassExpiredEventFaker.cs (hunk 2 above)
Comment (kamilbaczek) [reply Src/Fitnet.IntegrationTests/Offers/Preapare/PassExpiredEventFaker.cs]: Done :D

File: Src/Fitnet.IntegrationTests/Common/TestEngine/IntegrationEvents/Handlers/WebApplicationFactoryExtensions.cs (hunk 1 above)
Comment (kamilbaczek) [reply Src/Fitnet.IntegrationTests/Common/TestEngine/IntegrationEvents/Handlers/WebApplicationFactoryExtensions.cs]: Applied :D Thanks !

**Review by meaboutsoftware — APPROVED** [review]

**Pull request merged**